yunshin/4주차/BruteForce/Beakjoon/2309_일곱난장이.js

```diff
@@ -0,0 +1,23 @@
+// DFS
+const input = require('fs').readFileSync('yunshin/4주차/BruteForce/Beakjoon/input/b_2309.txt').toString().trim().split('\n')
+const visited = new Array(9).fill(false)
+
+const result = []
+function dfs(sumNum, depth) {
+  if (depth == 7 && sumNum == 100 && result.length == 0) {
+    for (let i = 0; i < 9; i++) if (visited[i]) result.push(Number(input[i]))
+  }
+
+  for (let i = 0; i < 9; i++) {
+    if (!visited[i]) {
+      visited[i] = true
+      dfs(sumNum + Number(input[i]), depth + 1)
+      visited[i] = false
+    }
+  }
+}
+dfs(0, 0)
+result.sort((a, b) => a - b)
+result.map((item) => {
+  console.log(item)
+})
```

Comment:
map은 보통 새로운 배열을 만들 때 사용하는 거 같아요 🙂 배열에 담아놓은 것들을 콘솔 명령어로 출력하고 싶다면

```suggestion
console.log(result.join('\n'))
```
은 어떨까요 ?! join은 split과 반대되고 합쳐주는 메서드입니다 :)